### ISSUE TITLE
[PANIC] peering.IsStaticallyPeered

### ISSUE THREAD

**Issue by viossat** [iotaledger/hornet#497]
I set a new 0.4.0 Hornet node up which crashed after a few minutes:
```
panic: runtime error: invalid memory address or nil pointer dereference 
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa328b5] 
goroutine 326 [running]: 
github.com/gohornet/hornet/pkg/peering.(*Manager).IsStaticallyPeered(0xc000246500, 0xc000e23a08, 0x1, 0x1, 0x3cf1, 0x0) 
	/src/pkg/peering/peering.go:174 +0x345 
github.com/gohornet/hornet/plugins/autopeering.configureEvents.func6(0xc006c3e090) 
	/src/plugins/autopeering/plugin.go:123 +0x398 
github.com/iotaledger/hive.go/autopeering/selection.peeringCaller(0xfe4e40, 0x1eff0a0, 0xc006c3e0a0, 0x1, 0x1) 
	/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20200520215850-06bf0dc5c0ef/autopeering/selection/events.go:43 +0x62 
github.com/iotaledger/hive.go/events.(*Event).Trigger(0xc000269800, 0xc006c3e0a0, 0x1, 0x1) 
	/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20200520215850-06bf0dc5c0ef/events/event.go:31 +0xdb 
github.com/iotaledger/hive.go/autopeering/selection.(*manager).triggerPeeringEvent(0xc0001a8280, 0xc005a0c100, 0xc005a0c100, 0x1) 
	/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20200520215850-06bf0dc5c0ef/autopeering/selection/manager.go:378 +0x5cb 
github.com/iotaledger/hive.go/autopeering/selection.(*manager).loop(0xc0001a8280) 
	/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20200520215850-06bf0dc5c0ef/autopeering/selection/manager.go:192 +0x3e5 
created by github.com/iotaledger/hive.go/autopeering/selection.(*manager).start 
	/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20200520215850-06bf0dc5c0ef/autopeering/selection/manager.go:84 +0x82 
```

**Comment by IoTMOD** [comment]
Should be fixed with #496

**Comment by viossat** [comment]
Sorry, I tried on the latest develop branch but didn't notice your very recent PR.
